bin/kytosd

```diff
@@ -55,7 +56,12 @@ if __name__ == '__main__':
     config = KytosConfig()
     controller = Controller(config.options['daemon'])
     if controller.options.foreground:
-        controller.start()
+
+        try:
+            controller.start()
+        except SystemExit:
+            sys.exit()
```

Comment:
Please print some useful information when exiting... for instance the reason of SystemExit exception.